CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # CHANGELOG
 
+### [v2.3 _(Jun 20, 2018)_](https://github.com/omise/omise-magento/releases/tag/v2.3)
+
+#### ✨ Highlights
+
+- Introducing support of Alipay payments (PR [#113](https://github.com/omise/omise-magento/pull/113))
+- Support for Multistore Magento 2 configuration (PR [#117](https://github.com/omise/omise-magento/pull/117))
+- Simplify plugin installation process by using composer (PR [#112](https://github.com/omise/omise-magento/pull/112))
+- Introduce Webhook feature  (PR [#107](https://github.com/omise/omise-magento/pull/107))
```

Comment:
Don't want to be too OCD but my eyes just spotted this 2 little spaces instead of 1 😆

Reply:
ok.. that has 0 influence on text visibility.. as it is formatted by wiki engine.. but of course.. will fix it..

Reply:
lol I know I know 😂